Week-3/autoLoan/style.css

```diff
@@ -0,0 +1,77 @@
+/* Set default fonts */
+body {
+    font-family: 'Roboto', 'Helvetica', 'sans-serif';
+  }
+  
+  /* Style the main heading */
+  h1 {
+    font-size: 16px;
+    color: #444444;
+    text-align: center;
+  }
+  
+  /* Define container class */
+  .container {
+    text-align: center;
+    margin: 5px;
+    font-size: 13px;
+  }
+  
+  /* Style each offer */
+  .offer {
+    display: inline-block;
+    width: 400px;
+    margin: 4px;
+    text-align: center;
+    border: 1px solid #cccccc;
+  }
+  
+  /* Style the offer title */
+  .offerTitle {
+    font-size: 14px;
+    font-weight: bold;
+    color: #ffffff;
+    background-color: #00b3b3;
+    padding: 15px;
+    margin: 0;
+  }
+  
+  /* Style the offer contents */
+  .offerContents {
+    padding: 20px;
+    background-color: #fafafa;
+  }
+  
+  /* Table styling */
+  table {
+    width: 100%;
+    background-color: #ffffff;
+    border-spacing: 0;
```

Comment:
It’s good to see the usage of `border-spacing` and` border-bottom` to handle table cell spacing, this can be more flexible than using border alone.